### PR TITLE
Amp carous(croll)el experiment - base PR

### PR DESCRIPTION
**Pull request by sriramkrish85**
cc : @cramforce @dvoytenko

### CODE REVIEW

**Comment by erwinmombay** [comment]
@sriramkrish85 LGTM

**Comment by jridgewell** [comment]
Oh, punny pun puns.

I don't see why we make these skeleton PRs, though.

**Comment by sriramkrish85** [comment]
-----------
:) 
To make reviews easier and get some progressive work done.

File: extensions/amp-carousel/0.1/amp-carousel.js [hunk 1]
```diff
@@ -25,6 +26,9 @@ class CarouselSelector {
         element.getAttribute('type') == 'slides') {
       return new AmpSlides(element);
     }
+    if (isExperimentOn(element.ownerDocument.defaultView, 'amp-carouscroll')) {
```
Comment (dvoytenko) [inline extensions/amp-carousel/0.1/amp-carousel.js]: This looks wrong :( We are reimplementing slides not scrolling carousel.

**Pull request merged**